package.json

```diff
@@ -51,6 +53,10 @@
     "axios": "^1.2.1",
     "fp-ts": "^2.13.1",
     "jsdom": "^20.0.3",
+    "next-swagger-doc": "^0.3.6",
+    "swagger-jsdoc": "^6.2.7",
+    "swagger-ui-dist": "^4.15.5",
+    "swagger-ui-react": "^4.15.5",
```

Comment:
don't think we need `next-swagger-doc` and `swagger-ui-react`

Reply:
yes that will be removed . It was just something I was trying